src/main/java/org/mozilla/jss/ssl/CipherPolicy.java

```diff
@@ -6,18 +6,21 @@
 
 public class CipherPolicy {
 
-    public static final CipherPolicy DOMESTIC =
-        new CipherPolicy(SocketBase.SSL_POLICY_DOMESTIC);
+public static final CipherPolicy DOMESTIC =
+	new CipherPolicy(SocketBase.SSL_POLICY_DOMESTIC);
```

Comment:
Looks like all fields and methods in the class lose their indentations, and the code is indented with tabs instead of spaces.

Reply:
There are some weird default choices with indentation, it is true.